docker/local/docker-compose.yml

```diff
@@ -16,8 +16,8 @@ services:
      - REDIS_REPLICATION_MODE=master
 
  turbinia-server:
-    #image: "turbinia-server-dev" # Use this for local development and comment out below line
-    image: "us-docker.pkg.dev/osdfir-registry/turbinia/release/turbinia-server:latest" # Latest stable
+    image: "turbinia-server-dev" # Use this for local development and comment out below line
```

Comment:
Unrelated changes?

Reply:
Changed back to online image, slipped in due to local testing.